src/ott/neural/layers.py

```diff
@@ -29,20 +29,17 @@ class PositiveDense(nn.Module):
   """A linear transformation using a weight matrix with all entries positive.
 
   Args:
-    dim_hidden: the number of output dim_hidden.
-    rectifier_fn: choice of rectifier function (default: softplus function).
-    inv_rectifier_fn: choice of inverse rectifier function
```

Comment:
trying to remember, but is there a reason we won't need the inverse map?

Reply:
This was introduces in here, for initialization: https://github.com/ott-jax/ott/pull/90